core/storage/skill/gae_models.py

```diff
@@ -220,7 +220,7 @@ class SkillSummaryModel(base_models.BaseModel):
     @staticmethod
     def get_deletion_policy():
         """Skill summary should be kept if associated skill is published."""
-        return base_models.DELETION_POLICY.KEEP_IF_PUBLIC
+        return base_models.DELETION_POLICY.LOCALLY_PSEUDONYMIZE
```

Comment:
These were required because we changed how skills work.

Reply:
Can you give more details about that?